CHANGELOG

```diff
@@ -1,3 +1,12 @@
+snowplow-web 0.9.0 (2022-07-19)
+---------------------------------------
+Add safety page_view_context deduplication for Redshift and Postgres (Close #104)
+Add support for Databricks users with Unity Catalog (Close #102)
```

Comment:
While you are here, can you please change the order here so that it follows the same order as the commits? For future reference let's try to order it in issue ascending number but it is fine for now, we are not strict on this.